src/RestApi.ts

```diff
@@ -133,10 +168,11 @@ export class RestApi {
      * @returns {Promise<Object>} - A promise that resolves with the user object, or rejects if an error occurs.
      */
     public getMyUser(): Promise<any> {
-        if (!this._myUserPromise) {
-            this._myUserPromise = this.get('user');
+        if (!this._user) {
+            // determineVersion will fill the user property
+            return this.determineApiVersion().then(() => this._user);
```

Comment:
if doRequest() already determines whether determineApiVersion is done, then getMyUser doesn't need to do the same, right?

Reply:
It doesn't necessarily, but without this code, `GET /user` will be requested twice: once from `determineApiVersion` and then again from `doRequest`.

Even though code-wise this is not ideal, I prefer not to have a useless REST request for all clients.

Reply:
I C. Ack!